src/doc/imagebufalgo.rst

```diff
@@ -1075,7 +1071,7 @@ Shuffling channels
 
     .. tabs::
   
-       .. code-tab:: c++
+       .. code-tab:: c++`
```

Comment:
Is this backtick a typo?... or perhaps there is something I'm not seeing something about needed syntax in .rst?

Thanks for the diff... Seeing everything should make mine quicker.

Reply:
Definitely a typo, thanks for checking, I'll remove it on the next pass!